Library/Formula/haproxy.rb

```diff
@@ -30,4 +29,8 @@ def install
     man1.install "doc/haproxy.1"
     bin.install "haproxy"
   end
+
+  test do
+    system bin/"haproxy", "-v"
```

Comment:
Probably not, sadly. I could look into setting up a proxy from port 8080 to 8081 or something briefly, but I'd probably have to write a configuration file and without a way to kill it after starting it, we may end up leaving the proxy running.